client/state/document-head/selectors.js

```diff
@@ -46,5 +46,8 @@ export function getFormattedTitle( state ) {
 		isSiteSection( state ) && getSiteTitle( state, getSelectedSiteId( state ) )
 	].filter( Boolean ).join( ' ‹ ' );
 
-	return `${ decodeEntities( title ) } — WordPress.com`;
+	return [
+		`${ decodeEntities( title ) }`,
```

Comment:
- The interpolation seems unnecessary since there's no other parts of the string
- Have we determined what the `decodeEntities` was intended for in the first place?
- The "WordPress.com" entry in the array is always truthy, so ever so slightly wasteful (in the other array, one, the other, or both entries could be falsey)

What about this instead?

```js
if ( title ) {
	title = decodeEntities( title ) + ' — ';
}

return title + 'WordPress.com';
```

Or, template-ified:

```js
if ( title ) {
	title = `${ decodeEntities( title ) } — `;
}

return `${ title }WordPress.com`;
```